services/ui-src/src/components/app/Error.test.tsx

```diff
@@ -5,9 +5,11 @@ import { testA11y } from "utils/testing/commonTests";
 const errorView = <Error />;
 
 describe("<Error />", () => {
-  test("Check that Error page renders", () => {
-    const { getByTestId } = render(errorView);
-    expect(getByTestId("error-view")).toBeVisible();
+  test("Check that Error message renders", () => {
+    const { getByRole } = render(errorView);
+    expect(getByRole("heading", { level: 2 })).toHaveTextContent(
+      "Sorry! An error has occurred."
+    );
```

Comment:
It's usually preferable to use `screen` for this exact reason: next time someone wants to add an assertion that uses a different `getBy` function, they won't also have to touch this line of code.

```suggestion
    render(errorView);
    expect(screen.getByRole("heading", { level: 2 })).toHaveTextContent(
      "Sorry! An error has occurred."
    );
```

Reply:
Lol that makes sense. It originally said `screen` wasn't available and that was my workaround and I just realized, I just needed to import it 💀